WcaOnRails/app/webpacker/components/EditEvents/Modals/EditQualificationModal/index.js

```diff
@@ -126,7 +126,7 @@ export default function EditQualificationModal({
       hasUnsavedChanges={hasUnsavedChanges()}
       disabled={disabled}
       tooltip={disabledReason}
-      triggerButtonProps={{ name: 'qualification'}}
+      triggerButtonProps={{ name: 'qualification' }}
```

Comment:
This (the word on the left) looks like a clickable element now, which I don't like.
![image](https://github.com/thewca/worldcubeassociation.org/assets/49137025/ba5c2742-521b-447a-9c3f-b3985d75a211)

Reply:
Thanks, fixed!